### PR TITLE
Choice matters.  A lot.

### PR DESCRIPTION
**Pull request by martinthomson**

### CODE REVIEW

**Comment by wkumari** [comment]
I agreed to change from DISCUSS to Abstain if this change was made - email thread for the curious / archives: https://mailarchive.ietf.org/arch/msg/ietf-and-github/vnXiskU2VGIx8VenEP34GWGfOwM/

A PR is close enough to integrated for me...

**Comment by mnot** [comment]
@wkumari I'm curious why this change is adequate; it's effectively just a rewording. It doesn't address the issue of the group choosing to adopt *the policies.*

**Review by chris-wood — APPROVED** [review]

**Comment by wkumari** [comment]
@mnot: Yes, it is just a rewording -- but (for me at least) it made it enough clearer that this was optional (WGs can choose to use GH, not WGs are expected to use GH) that I was willing to move to Abstain instead. 

The WG has consensus, and the charter sent us down the "GH FTW!" path - it isn't appropriate for me to block this just because *I* don't like it / think there is a better way of using RCS tooling / GH.  The process was followed, and the document is valuable - I don't like the tone, I don't like the implication, but it now falls into the "Well, that's just, like, your opinion, man..." territory.
(There isn't a technical objection, nor do I think it is specifically *dangerous* to the IETF / IETF stream, so...)

**Comment by mnot** [comment]
@wkumari - OK. This is made moot by the other changes that Martin made to the draft, but to be clear, I don't think that's a good line of argumentation.

The WG was charted to an explicit scope - that WGs had to opt into the practices it documented. If it _were_ producing a document that exceeded that scope (which is arguably did before the recent change referenced above), saying that the WG "has consensus" to violate its charter is meaningless, and letting that stand is damaging.

That's because charters allow people to decide whether or not they're going to follow a WG. If a WG can violate its charter and justify that with "but we had consensus to do so", charters are meaningless, and the check that requiring IETF consensus places upon WGs is seriously diminished.

File: draft-ietf-git-using-github.md [hunk 1]
```diff
@@ -59,7 +59,8 @@ specification.
 
 The main purpose of this document is providing guidelines for how Working Groups
 might integrate the capabilities provided by GitHub into their processes for
-developing Internet-Drafts.
+developing Internet-Drafts. The choice of whether to use GitHub or whether to
```
Comment (mnot) [inline draft-ietf-git-using-github.md]:
```suggestion
developing Internet-Drafts. The choices of whether to use GitHub and whether to
```

**Pull request merged**